test/e2e/addon/podidentityverify.go

```diff
@@ -3,26 +3,53 @@ package addon
 import (
 	"context"
 	_ "embed"
+	"fmt"
+	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go-v2/service/eks"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
 	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clientgo "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 
 	"github.com/aws/eks-hybrid/test/e2e/kubernetes"
 )
 
 const (
 	getAddonTimeout      = 2 * time.Minute
-	podIdentityAgent     = "eks-pod-identity-agent"
 	podIdentityDaemonSet = "eks-pod-identity-agent-hybrid"
+	podIdentityToken     = "eks-pod-identity-token"
+	policyName           = "pod-identity-association-role-policy"
+	PodIdentityS3Bucket  = "PodIdentityS3Bucket"
 )
 
+var PodIdentityS3BucketPrefix = strings.ToLower(PodIdentityS3Bucket)
```

Comment:
just define it as a constant pls, hardcode the value if you need t, it doesn't need to be derived